testapi.pm

```diff
@@ -553,6 +555,72 @@ sub wait_serial {
     return;    # false
 }
 
+=head2 wait_terminal
+
+  wait_terminal($pattern, timeout => 30, expect_not_found => 0, record_output => 0,
+                exclude_match => 0, no_regex => 0);
+
+This function is only valid when the console is set to a text terminal. It waits
```

Comment:
Because it probably won't be specific to one console, but many text based consoles. On the other hand it might fit into wait_serial and not be necessary at all. I will try to (re)move it.